### DISCUSSION TITLE
FFFTPのダークモード対応

### DISCUSSION THREAD

**Discussion by Nobuhiko-Muroi** [ffftp/ffftp#398] [Q&A]
FFFTPのダークモードを可能にしていただけますか？

**Comment by sayurin** [comment]
Windowsには旧来のGUI環境とWindows 8から始まったフラットデザインのGUI環境の２種類が存在します。
その上で、ダークモードは後者でのみサポートされているのに対し、ffftpは前者なので簡単にはダークモードを実現できません。

一応、こういうことをされている方もおられるようです。 https://github.com/ffftp/ffftp/issues/356

**Comment by Nobuhiko-Muroi** [comment]
御返事いただきまして誠にありがとうございます。

FFFTPも、仰る人のTablacus Explorerも導入済ですので、悩んでおります。

Tablacus Explorerのダークモードを設定、アドオンにて達成中の模様です。
